zerberus/zerberus.cpp

```diff
@@ -178,9 +178,7 @@ void Zerberus::play(const Ms::PlayEvent& event)
       {
       if (busy)
             return;
-      if (event.channel() >= MAX_CHANNEL)
```

Comment:
Can't happen anymore, MAX_CHANNELS being 256 now (and channel() being an unsigned char)

Reply:
And it fixes a warning for Linux and Mac too

Reply:
But shouldn't the code behave correctly regardless of the actual value of that parameter? If you propose to rely on `MAX_CHANNEL` being not less than (or equal to) a maximal value of `event.channel()`, could you please add the corresponding `static_assert` so that this code won't compile if the parameter's value (or, which is less likely, the return type of `event.channel()`) happens to change? Otherwise it is probably better to leave this check as it is, maybe suppressing the compiler warning in another way.

Reply:
static_assert ( MAX_CHANNEL == 255, "check range" );` ?

Reply:
Well, if testing exactly what we want to test then it should be something like
`static_assert(MAX_CHANNEL > std::numeric_limits<decltype(event.channel())>::max(), "need to add a check for a channel number range");`
(I don't know how to write the message better, maybe in this case the previous check should be actually left commented out nearby).

`MAX_CHANNEL` seems to be actually a number of channels rather than maximal channel number, maybe something like `MAX_CHANNELS` was actually meant by this name.

But such an assertion still leaves the same warning. However the following condition doesn't (but may potentially in the future as it is exactly the same).
`MAX_CHANNEL - 1 >= std::numeric_limits<decltype(event.channel())>::max()`

So, to sum up, maybe we can write an assertion in a way I proposed above and leave the previous check commented out. That check does the correct thing in a generic case, it is just the current parameter's value what makes it redundant.

Reply:
I had tried your 1st suggestion just to find out too that it doesn't help with getting rid of the warning. Will try that 2nd one now

Reply:
I can also change MAX_CHANNEL to MAX_CHANNELS if you want me to

Reply:
> I can also change MAX_CHANNEL to MAX_CHANNELS if you want me to

Yes, I believe it would be good to clarify the naming here, so it would be great if you could include it to this PR.